.github/workflows/win-wheels.yaml

```diff
@@ -22,63 +22,69 @@ jobs:
           - os: "windows-2019"
             arch: AMD64
             msvc_arch: x64
-            # windows requires windows-specific paths
-            vcpkg_cache: "c:\\vcpkg\\installed"
-            vcpkg_logs: "c:\\vcpkg\\buildtrees\\**\\*.log"
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        with:
-          repository: rasterio/rasterio
-          ref: ${{ env.REF }}
-
-      - name: Add to rasterio setup.cfg
-        shell: bash
-        run: |
-          echo -e "\n[build_ext]" >> setup.cfg
-          echo "include_dirs = C:/vcpkg/installed/x64-windows/include" >> setup.cfg
```

Comment:
I noticed this used to be capitalized here and it wasn't below. I am wondering if it is case sensitive.